resources/providers/gcplib/assets_inventory/assets.go

```diff
@@ -0,0 +1,68 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package assets_inventory
+
+import (
+	"cloud.google.com/go/asset/apiv1/assetpb"
+	"github.com/elastic/cloudbeat/resources/fetching"
+)
+
+var gcpBigQuery = []*assetpb.ListAssetsRequest{
+	{
+		// CIS Rules: 7.1
+		ContentType: assetpb.ContentType_IAM_POLICY,
+		AssetTypes:  []string{"bigquery.googleapis.com/Dataset"},
+	},
+}
+
+var gcpKms = []*assetpb.ListAssetsRequest{
+	{
+		// CIS Rules: 1.10
+		ContentType: assetpb.ContentType_RESOURCE,
+		AssetTypes:  []string{"cloudkms.googleapis.com/CryptoKey"},
+	},
+	{
+		// CIS Rules: 1.9
+		ContentType: assetpb.ContentType_IAM_POLICY,
+		AssetTypes:  []string{"cloudkms.googleapis.com/CryptoKey"},
+	},
+}
+
+var gcpIam = []*assetpb.ListAssetsRequest{
+	{
+		// CIS Rules: 1.6, 1.4
+		ContentType: assetpb.ContentType_IAM_POLICY,
```

Comment:
for simplicity, we can start with collection asset resource & iam_policy for all resources.

Reply:
you mean always providing `ContentType_IAM_POLICY` and `ContentType_RESOURCE`? seems wasteful if not needed, no?